TicketSystem/TicketSystem/Controllers/MessageController.cs

```diff
@@ -18,12 +20,11 @@ public MessageController(IMessageService messageService, ILogger<MessageControll
         }
 
         [HttpPost("{userId}")]
-        public async Task<Message> Post(int userId, Message message)
+        public async Task<Message> Post(int userId, Message message, CancellationToken cancellationToken)
         {
-            _logger.LogDebug("Post message bu userId {userId}", userId);
+            _logger.LogInformation("Post new message by userId {userId}, message: {JsonConvert.SerializeObject(message)}",userId, JsonConvert.SerializeObject(message));
```

Comment:
message: {JsonConvert.SerializeObject(message)} instead of this could live just message: {message}